### PR TITLE
Added Custom Assignment Cascade feature documentation.

### PR DESCRIPTION
**Pull request by MaceWindu1**
title: TALXIS Custom Assignment Cascade
tagline: Enables advanced assignment cascade configurations

### CODE REVIEW

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Entity
+
+## **TALXIS Configuration - Assignment Cascade**
+| Attribute name 	| Description 	| Type | Requirement
+|-	|-	|- |-
+| talxis_configuration_assignmentcascadeid 	| Unique id.	| Guid | System
+| talxis_filteringattributesavedstate 	| State of filtering attribute - pre-save or post-save.	| Two options | Optional
+| talxis_filterownershipattributename 	| Filtering attribute of type systemuser/team. 	| String | Optional
+| talxis_name 	| Name of the configuration. | String | Required
+| talxis_pluginstepid 	| Id of the associated sdk message processing step. | String (Guid) | Read-only
```
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: I would add "Autofills on save" or something like that so they know they don't have to do anything with this

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| talxis_recordset 	| Fetch xml filter query. The logic will apply only to these records. Default query is generated after the record is saved.	| String | Optional
+| talxis_relationshipname 	| Logical name of entity relationship. This one-to-many relationship must exist between the source and the target entity. 	| String | Required
+| talxis_sourceentityname 	| Logical name of the source (triggering) entity.| String | Read-only
+| talxis_sourceownershipattributename 	| Logical name of the source (triggering) attribute of type systemuser/team. Leave empty to use default value = 'ownerid'.	| String | Required
+| talxis_targetentityname 	| Logical name of the target entity.	| String | Read-only
+| talxis_targetownershipattributename 	| Logical name of the target attribute of type systemuser/team. Leave empty to use default value = 'ownerid'. 	| String | Required
+| StateCode 	| Entity state. Values are Active/Inactive. 	| State | System
```
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:30]: I might also write that inactive records means the logic does not trigger

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+| StatusCode 	| Status of the entity state. Values are Active/Inactive.	| Status | System
+
+<br>
+![Configuration Main Form](/.attachments/AssignmentCascade/AssignmentCascadeConfigurationMainForm.png)
+<br><br>
+---
+
+# Code - Configuration
```
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:38]: And maybe add a sample of execution

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md (hunk 1 above)
Comment (MaceWindu1) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: I think it's obvious read-only fields should be left untouched.

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md (hunk 2 above)
Comment (MaceWindu1) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:30]: Good point.

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md (hunk 1 above)
Comment (JanKrupicka) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: Sure Im ok with that.

**Comment by MaceWindu1** [comment]
Waiting for review from @honzakostejn.

**Review by honzakostejn — APPROVED** [review]

**Pull request merged**